tools/sdg/topics/main.py

```diff
@@ -161,6 +163,25 @@ def generate_full():
   nodes.sort(key=lambda x: x['dcid'])
   with open(_SDG_TOPIC_JSON, 'w') as fp:
     json.dump({'nodes': nodes}, fp, indent=2)
+  generate_mcf(nodes)
+
+
+def generate_mcf(nodes: list[dict]) -> None:
+  logging.info("Writing MCF to: %s", _MCF_PATH)
+  os.makedirs(_TMP_DIR, exist_ok=True)
+  with open(_MCF_PATH, 'w') as out:
+    for node in nodes:
+      out.write(generate_mcf_node(node))
+
+
+def generate_mcf_node(node: dict) -> str:
+  return f"""Node: dcid:{node['dcid'][0]}
+dcid: "{node['dcid'][0]}"
+relevantVariable: "{','.join(node['relevantVariableList'])}"
+name: "{node['name'][0]}"
+typeOf: dcid:{node['typeOf'][0]}
+
```

Comment:
```
relevantVariable: "sdg/AG_FOOD_WST,dc/topic/sdgAGFOODWST_sdgfoodWasteSector"
```

```
relevantVariable: dcid:sdg/AG_FOOD_WST, dcid:dc/topic/sdgAGFOODWST_sdgfoodWasteSector
```

There's an important difference here --- first one is a single string, second one is a list of references!

Reply:
Thanks - still grokking MCF! Done. MCF in CNS is updated.